### PR TITLE
[FIX] Colab-flag

### PR DESCRIPTION
**Pull request by elephaint**
- Adds Colab flags to all notebooks
- Adds Colab badge to capabilities notebooks

### CODE REVIEW

**Review by AzulGarza — CHANGES_REQUESTED** [review]
thank you so much @elephaint, it seems it involved a lot of effort. i'm very sorry about asking a change to his pr, but would you be open to implementing the following logic? i think it can improve readability and might be easier to maintain:

- define `IN_COLAB = 'google.colab' in sys.modules` in `nixtla/utils` so we can import it instead of defining each time.
- import all the required deps for colab in one cell:

```
#| hide
if not IN_COLAB:
    from nixtla.utils import colab_badge
    rom dotenv import load_dotenv

**Comment by elephaint** [comment]
> thank you so much @elephaint, it seems it involved a lot of effort. i'm very sorry about asking a change to his pr, but would you be open to implementing the following logic? i think it can improve readability and might be easier to maintain:
> 
> * define `IN_COLAB = 'google.colab' in sys.modules` in `nixtla/utils` so we can import it instead of defining each time.
> * import all the required deps for colab in one cell:
> 
> ```
> #| hide
> if not IN_COLAB:
>     from nixtla.utils import colab_badge
>     rom dotenv import load_dotenv
> ```

@AzulGarza in Colab nixtla is not installed, so we can't import IN_COLAB when it's part of nixtla. Unless the first line in Colab is `%pip install nixtla', but then we can't make that install conditional on being in Colab.

File: lightning_logs/version_4/events.out.tfevents.1715239615.ospc
Comment (AzulGarza) [inline lightning_logs/version_4/events.out.tfevents.1715239615.ospc:1]: @elephaint could you please remove these files? i think they were pushed by mistake. :)

**Review by AzulGarza — CHANGES_REQUESTED** [review]

File: lightning_logs/version_4/events.out.tfevents.1715239615.ospc
Comment (elephaint) [reply lightning_logs/version_4/events.out.tfevents.1715239615.ospc:1]: Yes, indeed...

**Review by AzulGarza — APPROVED** [review]

**Pull request merged**